tests/test_permissions.py

```diff
@@ -0,0 +1,186 @@
+import json
+from schematools.importer.ndjson import NDJSONImporter
+from pg_grant import query
+from schematools.permissions import create_acl_from_profiles, apply_schema_and_profile_permissions
+from sqlalchemy.exc import ProgrammingError
+import pytest
+from schematools.types import DatasetSchema
```

Comment:
even `isort` draaien zou hier geen kwaad kunnen. (of in je editor de imports laten ordenen)